setup.py

```diff
@@ -1,120 +1,146 @@
+        'include_files': ['res',
+                          'RELEASE-VERSION',
+                          ('lib/uid.dll', 'uid.dll'),
+                          ('lib/qt.conf', 'qt.conf'),
+                          ('lib/xdelta3.exe', 'xdelta3.exe'),
+                          ('lib/lua51.dll', 'lua51.dll')],
+        'icon': 'res/faf.ico',
+        'include_msvcr': False,
+        'optimize': 2,
+        'packages': ['cffi', 'pycparser', 'PyQt4', 'PyQt4.uic',
+                     'PyQt4.QtGui', 'PyQt4.QtNetwork', 'win32com', 'win32com.client'],
+        'silent': True,
+        'excludes': ['numpy', 'scipy', 'matplotlib', 'tcl', 'Tkinter']
+    }
+
+    shortcut_table = [
+        ('DesktopShortcut',           # Shortcut
+         'DesktopFolder',             # Directory_
+         'FA Forever',                # Name
+         'TARGETDIR',                 # Component_
+         '[TARGETDIR]FAForever.exe',  # Target
+         None,                        # Arguments
+         None,                        # Description
+         None,                        # Hotkey
+         None,                        # Icon
+         None,                        # IconIndex
+         None,                        # ShowCmd
+         'TARGETDIR'                  # WkDir
+         )
+    ]
+
+    target_dir = '[ProgramFilesFolder][ProductName]'
+    upgrade_code = '{ADE2A55B-834C-4D8D-A071-7A91A3A266B7}'
+
+    if False:  # Beta build
+        product_name += " Beta"
+        upgrade_code = '{2A336240-1D51-4726-B36f-78B998DD3740}'
+
+    bdist_msi_options = {
+        'upgrade_code': upgrade_code,
+        'initial_target_dir': target_dir,
+        'add_to_path': False,
+        'data': {'Shortcut': shortcut_table},
+    }
+
+    # GUI applications require a different base on Windows (the default is for a
+    # console application).
+    base = None
+    if sys.platform == 'win32':
+        base = 'Win32GUI'
+
+    exe = Executable(
+        'src/__main__.py',
+        base=base,
+        targetName='FAForever.exe' if sys.platform == 'win32' else 'FAForever',
+        icon='res/faf.ico',
+        includes=[os.path.join(os.path.dirname(PyQt4.uic.__file__), "widget-plugins"),
+                "PyQt4.uic.widget-plugins"]
+    )
+
+    setup(
+        name=product_name,
+        version=msi_version,
+        description=description,
+        long_description=long_description,
+        author=author,
+        maintainer=maintainer,
+        url=url,
+        license=license,
+        options={'build_exe': build_exe_options,
+                 'bdist_msi': bdist_msi_options},
+        executables=[exe],
+        requires=['bsdiff4', 'sip', 'PyQt4', 'cx_Freeze', 'cffi', 'py', 'faftools'],
+    )
+else:
+    from distutils.core import setup
+    from setuptools import find_packages
+    product_name = 'fafclient'
+    packages=find_packages()
+    print(packages)
+    setup(
+        name=product_name,
+        version=os.getenv('FAFCLIENT_VERSION'),
```

Comment:
All right; I think we want to get rid of all of the other version information and simply provide it externally through env during build. This means the build script should be responsible for writing RELEASE-VERSION directly, and the 'live' code should always read this.

How does that sound?